php/libraries/Module.class.inc

```diff
@@ -76,7 +76,11 @@ class Module extends \LORIS\Router\PrefixRouter implements RequestHandlerInterfa
                     . strtolower(substr($class, strlen("LORIS\\$name\\")))
                     . ".class.inc";
                     if (!file_exists($fpath)) {
-                        throw new \NotFound($name);
+                        throw new \LorisException(
+                            "Could not resolve $fpath to a LORIS module. " .
+                            "Note that PHP filenames within modules must be " .
+                            "entirely lowercase."
+                        );
```

Comment:
This is both an assumption about what the problem is, and a much less specific exception than was previously thrown.

Reply:
In my example when a file was named `MediaFileDownloadValidator.class.inc` the error generated was `Uncaught NotFound: media` even though `media` itself was fine. I had to read the source code and change the original file to `mediafiledownloadvalidator` to move forward. The exception text didn't help at all and was very misleading.

Can you suggest something you'd prefer?

Reply:
The problem is that it couldn't find the module. In *your* case that happened to be because of case sensitivity, what happens in all the other situations that can cause a file_exists check to fail?

Reply:
I can remove the part about modules needing to be lowercase. Are you fine with the rest of it?

Reply:
As discussed in person, will change this error to display both the name and the fpath